moto/cloudfront/urls.py

```diff
@@ -1,13 +1,16 @@
 """cloudfront base URL and path."""
 from .responses import CloudFrontResponse
 
+
 response = CloudFrontResponse()
 
 url_bases = [
     r"https?://cloudfront\.amazonaws\.com",
 ]
-
+# import pytest
```

Comment:
We don't need to keep this

Reply:
Hi @bblommers,

Sorry about those commented out lines, I missed them in my cleanup...

I'm happy to write additional tests, but are you referring to my existing case and breaking that up into 3 different tests, or writing additional tests to cover scenarios not yet covered? If the latter, what scenarios did you have in mind?

Thanks.
Paul

Reply:
Oh, Never mind, I just looked at models.py, and now I realize you're asking to cover these errors:

```
        if Id not in self.distributions or Id is None:
            raise NoSuchDistribution
        if not IfMatch:
            raise InvalidIfMatchVersion
        if not DistributionConfig:
            raise NoSuchDistribution
```

I'll write those up.